src/runtime/crt/host/microtvm_api_server.py

```diff
@@ -162,13 +161,10 @@ def generate_project(self, model_library_format_path, standalone_crt_dir, projec
         )
 
     def build(self, options):
-        args = ["make"]
-        if options.get("verbose"):
-            args.append("VERBOSE=1")
-
-        args.append(self.BUILD_TARGET)
-
-        subprocess.check_call(args, cwd=PROJECT_DIR)
+        build_dir = PROJECT_DIR / "build"
+        build_dir.mkdir()
+        subprocess.check_call(["cmake", ".."], cwd=build_dir)
+        subprocess.check_call(["make", "-j12"], cwd=build_dir)
```

Comment:
why -j12? we didn't have this one before

Reply:
totally missed that, fixed now!